### ISSUE TITLE
[redesign] Add note to typography section regarding minimum font size and weight for accessibility

### ISSUE THREAD

**Issue by mansona** [ember-learn/ember-styleguide#211]
at the top of https://deploy-preview-145--ember-styleguide.netlify.com/concepts/typeography refer to https://deploy-preview-145--ember-styleguide.netlify.com/concepts/colours and make sure that the contrast ratio (FAIL, AA and AAA) is described correctly
